tests/helpers.py

```diff
@@ -2370,3 +2366,33 @@ def get_pv_size(storageclass=None):
         if pv_obj['spec']['storageClassName'] == storageclass:
             return_list.append(pv_obj['spec']['capacity']['storage'])
     return return_list
+
+
+def get_cluster_proxies():
+    """
+    Get http and https proxy configuration.
+    * If configuration ENV_DATA['http_proxy'] (and prospectively
+        ENV_DATA['https_proxy']) exists, return the respective values.
+        (If https_proxy not defined, use value from http_proxy.)
+    * If configuration ENV_DATA['http_proxy'] doesn't exist, try to gather
+        cluster wide proxy configuration.
+    * If no proxy configuration found, return empty string for both http_proxy
+        and https_proxy.
+
+    Returns:
+        tuple: (http_proxy, https_proxy)
+
+    """
+    if 'http_proxy' in config.ENV_DATA:
+        http_proxy = config.ENV_DATA['http_proxy']
+        https_proxy = config.ENV_DATA.get(
+            'https_proxy', config.ENV_DATA['http_proxy']
+        )
+    else:
+        ocp_obj = ocp.OCP(kind=constants.PROXY, resource_name='cluster')
+        proxy_obj = ocp_obj.get()
+        http_proxy = proxy_obj['spec'].get('httpProxy', '')
```

Comment:
Based on our offline conversation where I have asked about if we will have Proxy object even on non proxy env I've got answer from Daniel that he found in documentation:

```
When a cluster is installed or upgraded without the proxy configured, a Proxy object is still generated but it will have a nil spec.
```

So my understanding is that spec can be None in python, so to be on safe side I would rather go with something like:
```suggestion
        http_proxy = proxy_obj.get('spec', {}).get('httpProxy', '')
```

The same below.

Otherwise LGTM

Reply:
Fixed by 166c156.

Reply:
Spec can't be `None`, neither it can be removed from the CRD.

Reply:
@mbukatov thanks for checking, this way it is safe...